v2/pkg/internal/epoller/README.md

```diff
@@ -0,0 +1,8 @@
+# Epoller
```

Comment:
If we copy paste the epoll implementation to the engine, we should import it from the router to avoid maintaining the same implementation twice. I commented it on the cosmo PR.